pkgs/tools/misc/hexbinhex/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "hexbinhex";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "dj-on-github";
+    repo = "${pname}";
```

Comment:
sometimes pname is overriden, so please keep `repo = "hexbinhex";` to avoid unwanted propagation of overriden pnames

Reply:
I have changed it to what you suggested.